### ISSUE TITLE
icerec, simulation - remove dst-extractor from meta-projects (Trac #1614)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1606]
according to #1571 `dst-extractor` is no more

Migrated from https://code.icecube.wisc.edu/ticket/1614
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:42", 
    "description": "according to #1579 `dst-extractor` is no more", 
    "reporter": "nega", 
    "cc": "olivas", 
    "resolution": "fixed", 
    "_ts": "1550067102580394", 
    "component": "cmake", 
    "summary": "icerec, simulation - remove dst-extractor from meta-projects", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-03-30T21:00:35", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160420T18:52:16: olivas changed milestone from "March 2016" to "April 2016"

**Comment by icecube-trac** [comment]
Trac update at 20160422T14:25:13: 
* nega commented:

> In changeset 144987:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="144987"
> remove dst-extractor
> 
> closes #1606
> ```


* nega changed status from "new" to "closed"
* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:42: 
* olivas changed milestone from "April 2016" to ""
* olivas commented:

> Milestone April 2016 deleted
